### PR TITLE
Update bitcoin.conf with example for pruning

### PR DESCRIPTION
**Pull request by coinables**
This option is becoming more popular recently, and I propose an example to be shown in the bitcoin.conf.

### CODE REVIEW

**Comment by laanwj** [comment]
Concept ACK on adding the example.

Explicitly mentioning 3GB is not a good idea. These numbers peppered around the documentation tend to get outdated sooner than later. The [size of the UTXO set](http://statoshi.info/dashboard/db/unspent-transaction-output-set) is growing every day, and that can't be pruned.

File: contrib/debian/examples/bitcoin.conf [hunk 1]
```diff
@@ -131,6 +131,10 @@
 # be validated sooner.
 #paytxfee=0.00
 
+# Prune your node to the last n blocks. 550 is the minimum. 
+# Pruning can drastically reduce the amount of disk space required to run bitcoin down to ~3GB.
```
Comment (laanwj) [inline contrib/debian/examples/bitcoin.conf]: You have the unit wrong, it's not in blocks but in MiB. Maybe you can borrow some description text from the help message:
```
  -prune=<n>
       Reduce storage requirements by enabling pruning (deleting) of old
       blocks. This allows the pruneblockchain RPC to be called to
       delete specific blocks, and enables automatic pruning of old
       blocks if a target size in MiB is provided. This mode is
       incompatible with -txindex and -rescan. Warning: Reverting this
       setting requires re-downloading the entire blockchain. (default:
       0 = disable pruning blocks, 1 = allow manual pruning via RPC,
       >550 = automatically prune block files to stay under the
       specified target size in MiB)
```

File: contrib/debian/examples/bitcoin.conf [hunk 2]
```diff
@@ -131,8 +131,10 @@
 # be validated sooner.
 #paytxfee=0.00
 
-# Prune your node to the last n blocks. 550 is the minimum. 
-# Pruning can drastically reduce the amount of disk space required to run bitcoin down to ~3GB.
+# Enable pruning to reduce storage requirments by deleting old blocks.
```
Comment (paveljanik) [inline contrib/debian/examples/bitcoin.conf]: requirements

File: contrib/debian/examples/bitcoin.conf [hunk 3]
```diff
@@ -131,8 +131,10 @@
 # be validated sooner.
 #paytxfee=0.00
 
-# Prune your node to the last n blocks. 550 is the minimum. 
-# Pruning can drastically reduce the amount of disk space required to run bitcoin down to ~3GB.
+# Enable pruning to reduce storage requirments by deleting old blocks. 
+# Value is in MiB. 550 is the minimum.
```
Comment (paveljanik) [inline contrib/debian/examples/bitcoin.conf]: 550 is the minimum, but you describe entering 1...

File: contrib/debian/examples/bitcoin.conf [hunk 4]
```diff
@@ -131,8 +131,10 @@
 # be validated sooner.
 #paytxfee=0.00
 
-# Prune your node to the last n blocks. 550 is the minimum. 
-# Pruning can drastically reduce the amount of disk space required to run bitcoin down to ~3GB.
+# Enable pruning to reduce storage requirments by deleting old blocks. 
+# Value is in MiB. 550 is the minimum. 
+# This mode is incompatible with -txindex and -rescan.
+# default = 0, 1 = allows manual pruning via RPC, >550 = target to stay under in MiB.
```
Comment (paveljanik) [inline contrib/debian/examples/bitcoin.conf]: >=550

**Review by paveljanik — CHANGES_REQUESTED** [review]

**Comment by sipa** [comment]
ACK after squashing.

File: contrib/debian/examples/bitcoin.conf [hunk 5]
```diff
@@ -131,6 +131,13 @@
 # be validated sooner.
 #paytxfee=0.00
 
+# Enable pruning to reduce storage requirments by deleting old blocks. 
+# This mode is incompatible with -txindex and -rescan.
+# 0 = default(no pruning).
```
Comment (paveljanik) [inline contrib/debian/examples/bitcoin.conf]: space before "(". And please squash.

File: contrib/debian/examples/bitcoin.conf [hunk 6]
```diff
@@ -131,6 +131,13 @@
 # be validated sooner.
 #paytxfee=0.00
 
+# Enable pruning to reduce storage requirments by deleting old blocks.
```
Comment (paveljanik) [inline contrib/debian/examples/bitcoin.conf]: requirments -> requirements

**Review by paveljanik — CHANGES_REQUESTED** [review]

**Comment by fanquake** [comment]
utACK c0651cc

**Comment by laanwj** [comment]
I think we should move this file somewhere else - `doc/examples` maybe? After all it is a general example of a `bitcoin.conf` and there is nothing debian-specific to it.

**Pull request merged**